moduleframework/common.py

```diff
@@ -231,3 +192,253 @@ def sanitize_cmd(cmd):
         if char in cmd:
             cmd = cmd.replace(char, '\\'.join(char))
     return cmd
+
+
+def get_profile():
+    """
+    Return profile name string
+
+    :return: str
+    """
+    amodule = os.environ.get('PROFILE')
+    if not amodule:
+        amodule = "default"
+    return amodule
+
+
+def get_url():
+    """
+    Return actual URL if overwritten by
+    env variable "URL"
+
+    It redefines location of testing subject
+
+    :return:
+    """
+    amodule = os.environ.get('URL')
+    return amodule
+
+
+def get_config():
+    """
+    Read the module's configuration file
+
+    :default: ``./config.yaml`` in the ``tests`` directory of the module's root directory
+    :envvar: **CONFIG=path/to/file** overrides default value.
+    :return: str
+    """
+    cfgfile = os.environ.get('CONFIG') or './config.yaml'
+    try:
+       with open(cfgfile, 'r') as ymlfile:
+           xcfg = yaml.load(ymlfile.read())
+           return xcfg
+    except IOError:
+       raise ConfigExc(
+           "Error: File '%s' doesn't appear to exist or it's not a YAML file." %
+            cfgfile + " " +
+           "Tip: If the CONFIG envvar is not set, mtf-generator looks for './config'.")
+
+
+def get_compose_url():
+    """
+    Return Compose Url if set in config or via
```

Comment:
Return a Compose Url. If a ***COMPOSEURL* envvar is not set, 
it's taken from a ``config.yaml`` file.